src/app/components/header-nav/header-nav.component.html

```diff
@@ -0,0 +1,6 @@
+
+    <nav class="menu">
+        <img src="http://assets.stickpng.com/images/58f37720a4fa116215a9240f.png" alt="Rick_and_morty_image" class ="image ">
```

Comment:
Download the image and use locally (Save on the asstes/images)
Also and an alt property